README.md

```diff
@@ -130,6 +130,26 @@ const ipld = new Ipld({
 })
 ```
 
+##### `options.loadFormat(codec, callback)`
+
+| Type | Default |
+|------|---------|
+| `Function` | `null` |
+
+Function to dynamically load an [IPLD Format](https://github.com/ipld/interface-ipld-format). It is passed a string `codec`, the multicodec of the IPLD format to load and a callback function to call when the format has been loaded. e.g.
+
+```js
+const ipld = new Ipld({
+  loadFormat (codec, callback) {
```

Comment:
Wow, I didn't know that this syntax is a things. It's nice to learn new things by code reviewing.